cmd/wallet-web/src/components/FlyoutMenu/FlyoutMenu.vue

```diff
@@ -1,11 +1,12 @@
+<!--
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
 <template>
   <div class="h-11">
     <!--Todo make flyout icon and text configurable -->
-    <button
-      v-if="type === 'default'"
-      id="default"
-      class="inline-flex items-center py-2 px-3 w-screen md:w-auto"
-    >
+    <button v-if="type === 'default'" class="inline-flex items-center py-2 px-3 w-screen md:w-auto">
```

Comment:
Should we also set `id`s to `default` and `vault` flyout menus, since we add it as a prop?

Reply:
will put some thought into it and address in a separate pr  created issue #1288